packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -228,6 +230,7 @@ export async function getConsolidated({
   };
   try {
     if (!bundle) {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
should we just create a new local variable that points to `bundle` so we don't need these?